### PR TITLE
EPMUII-99: Extend BE response for displaying users ranks

### PR DESCRIPTION
**Pull request by Shutya**

### CODE REVIEW

File: app/libs/commonCrudOperations.js [hunk 1]
```diff
@@ -9,6 +9,16 @@ export const getAll = (Model) => {
   };
 };
 
+export const getAllAndSort = (Model, sortSettings) => {
```
Comment (VladCherniavsky) [inline app/libs/commonCrudOperations.js]: What about using `findAndCount` static method of the model ? You can see implementation [here](https://github.com/FrontSpot-Community/code-verify/blob/master/app/libs/mongooseExtensionMethods.js).  Third parameter is sorting setting

Comment (Shutya) [reply app/libs/commonCrudOperations.js]: Fixed. Thank you.

File: app/libs/countUserSolutions.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import mongoose from 'mongoose';
+import config from '../configuration';
+import solution from '../models/solution';
+import user from '../models/user';
+
+mongoose.connect(config.get('db:connection'));
+mongoose.Promise = Promise;
```
Comment (VladCherniavsky) [inline app/libs/countUserSolutions.js]: I think it will be better to reuse `libs/mongoose`.

**Comment by VladCherniavsky** [comment]
ок

**Review by VladCherniavsky — APPROVED** [review]

**Review by spzm — APPROVED** [review]

**Pull request merged**